### ISSUE TITLE
1 Column custom symbol for Hz

### ISSUE THREAD

**Issue by trstephen** [trstephen/ceng355#4]
Write a custom symbol to the LCD table that displays `Hz` in one character space.

**Comment by trstephen** [comment]
[This thing](https://omerk.github.io/lcdchargen/) spits out the byte array for a custom character. Still have to write the uploading code but the designer is nice.

**Comment by trstephen** [comment]
Apparently, this

``` c
byte customChar[8] = {
    0b10100,
    0b11100,
    0b10100,
    0b01111,
    0b00010,
    0b00100,
    0b01111,
    0b00000
};
```

will generate this
<img width="140" alt="hz" src="https://cloud.githubusercontent.com/assets/6734837/20153706/d94e0eea-a678-11e6-879f-be4c9eb7bbeb.png">
@tkun3 Can you come up with a better Hz symbol? This one has a weird letter weight IMO

**Comment by trstephen** [comment]
Implemented in #25
